docs/testbed/README.testbed.ocs_data_plan_deployment.md

```diff
@@ -0,0 +1,262 @@
+# OCS TestBed Setup
+This document describes the steps to setup an optical circuit switch(OCS) testbed, delpoy OCS to the sonic-mgmt test framework for scripts running.
+
+## Prepare Testbed host
+According to testbed.Setup[testbed.Setup], prepare a PC with Ubuntu 20.04(recommanded), install the necessary libraries, and download sonic-mgmt docker container for test scripts running as well as a PTF container for service distribution.
+
+1. Install Ubuntu AMD64 on your host or VM
+    - The host PC needs to have at least 20GB of memory free
+    - If the testbed host is a VM, then it must support nested virtualization
+        - [Instructions for Hyper-V based VMs](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/nested-virtualization#configure-nested-virtualization)
+2. Prepare your environment based on Ubuntu
+    The Ubuntu version now is recommanded with **Ubuntu 20.04**
```

Comment:
Ubuntu 20.04 is end of support as of 2025. The minimum Ubuntu version is 22.04 or 24.04.